pom.xml

```diff
@@ -527,6 +528,20 @@
                 <artifactId>maven-dependency-plugin</artifactId>
                 <version>3.3.0</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.1</version>
```

Comment:
I think `5.1.8` is the most recent version of this plugin? Any reason to use an older version?

Also, do we need to add an `<extensions>true</extensions>` statement here? I've seen it in other projects, not sure exactly what it does though?

Reply:
I have compared the output in the manifest with and without the `extensions` tag and there are no differences.